adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -266,6 +256,28 @@ public void export(MissingRequiredFileNotice toExport) throws IOException {
                 .writeTo(streamGenerator.getStream());
     }
 
+    @Override
+    public void export(EmptyFileErrorNotice toExport) throws IOException {
```

Comment:
Reading it, thinking it should be factorized as the only differing line concerns `.setSeverity(`